comb_spec_searcher/strategies/strategy_pack.py

```diff
@@ -266,6 +303,30 @@ def add_verification(
             iterative=self.iterative,
         )
 
+    def remove_strategy(self, strategy: CSSstrategy):
+        """Create a new pack where the strategy is removed."""
+        d = strategy.to_jsonable()
+
+        def replace_list(strats):
+            """Return a new list with the replaced fusion strat."""
```

Comment:
Docstring needs to be updated

Reply:
True! That was copied over from the tilescope branch as well, woops